.travis.yml

```diff
@@ -44,6 +47,9 @@ matrix:
     - php: 7.3
       env:
         - DEPS=latest
+    - php: 8.0
```

Comment:
Please double check #2' checklist and compare with the current matrix.
For any version, we have `DEPS=lowest` and `DEPS=latest`.

Since `8.0` is not yet available on travis, we have to use `nightly` instead.
As we are dropping PHP <7.3 in `composer.json`, please also drop old versions from the matrix and ensure that *all* supported versions are being tested by travis.

This includes:
- PHP 7.3
- PHP 7.4
- PHP 8.0 (`nightly` in travis for now)